.github/workflows/ci.yml

```diff
@@ -54,5 +58,19 @@ jobs:
                 adduser -u $(id -u) -G build -D -H build &&
                 su build -c 'cargo run -p package -- dist'
               "
+      - name: Build & Package
+        if: ${{ matrix.target == 'armv7l' }}
+        run: |
+          docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
+          docker build . \
+            -f Dockerfile.armv7l \
+            -t jump/armv7l \
+            --build-arg GROUP="$(id -g)" \
+            --build-arg USER="$(id -u)"
+          docker cp $(docker create jump/armv7l):/code/dist .
       - name: Integration Tests
+        if: ${{ matrix.target != 'armv7l' }}
         run: examples/run.sh --no-package
+      - name: Integration Tests
+        if: ${{ matrix.target == 'armv7l' }}
+        run: setarch armv7l examples/run.sh --no-package
```

Comment:
This is the only thing I have not been able to test, as I do not have access to the Linux ARM runners in the fork. Would be great if you can approve the CI to run, so I can verify that the integration tests works :)

Reply:
I got this working on the Ubuntu-22.04 runner.